reactapp/package.json

```diff
@@ -13,32 +13,33 @@
   "private": true,
   "proxy": "https://demo.hyva.io",
   "dependencies": {
-    "@heroicons/react": "^1.0.3",
+    "@heroicons/react": "^1.0.1",
+    "@hyva/react-checkout": "file:../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/reactapp/src",
```

Comment:
Seems the path has changed to `hyva-themes/magento2-react-checkout/src/reactapp/` if i'm correct.

Reply:
@rlaven no, `src` is present there. it is actually the correct path.

Reply:
@rlaven also please note, this work is now moved to https://github.com/hyva-themes/magento2-checkout-example officially.

Reply:
@rlaven What he says. I'm going to deprecate this repository, because all work has been moved to the repo @rajeev-k-tomy mentioned